contracts/SportMarkets/SportPositions/SportPositionalMarket.sol

```diff
@@ -137,6 +140,7 @@ contract SportPositionalMarket is OwnedWithInit, ISportPositionalMarket {
             _mint(creator, initialMint);
         }
 
+        safeBoxImpact = ISportsAMM(sportsAMM).safeBoxImpact();
```

Comment:
why do we need to store this as local variable? Is it in case SB impact changes?

Reply:
yes, in case we change it after the market is created